regression-test/suites/schema_change_p0/test_add_keys_light_schema_change.groovy

```diff
@@ -0,0 +1,454 @@
+        qt_31_duplicate_multi_rowset """ SELECT * FROM add_keys_light_schema_change WHERE new_key_column = 1; """
+
+        (code, out, err) = be_run_full_compaction(backendId_to_backendIP.get(backend_id), backendId_to_backendHttpPort.get(backend_id), table_tablets[0][0])
+        logger.info("Run compaction: code=" + code + ", out=" + out + ", err=" + err)
+        assertEquals(code, 0)
+
+        qt_31_duplicate_compaction """ SELECT * FROM add_keys_light_schema_change; """
+        qt_31_duplicate_compaction """ SELECT * FROM add_keys_light_schema_change WHERE new_key_column = 2; """
+        qt_31_duplicate_compaction """ SELECT * FROM add_keys_light_schema_change WHERE new_key_column = 1; """
+
+        // case 4.1: light schema change unique : with multi version rowset and compaction
+        sql """ DROP TABLE IF EXISTS add_keys_light_schema_change """
+        sql """
+                CREATE TABLE IF NOT EXISTS add_keys_light_schema_change (
+                    `user_id` LARGEINT NOT NULL COMMENT "用户id",
+                    `username` VARCHAR(50) NOT NULL COMMENT "用户昵称",
+                    `city` VARCHAR(20) COMMENT "用户所在城市",
+                    `age` SMALLINT COMMENT "用户年龄",
+                    `sex` TINYINT COMMENT "用户性别",
+                    `phone` LARGEINT COMMENT "用户电话",
+                    `address` VARCHAR(500) COMMENT "用户地址",
+                    `register_time` DATETIME COMMENT "用户注册时间")
+                UNIQUE KEY(`user_id`, `username`, `city`)
+                DISTRIBUTED BY HASH(`user_id`)
+                BUCKETS 4
+                PROPERTIES ( "replication_num" = "1", "light_schema_change" = "true");
+            """
+        sql """ 
+                INSERT INTO add_keys_light_schema_change VALUES
+                (1, 'Jone', 'Beijing', 10, 1, 10010, 'Haidian', '2017-10-01 12:00:00')
+            """
+        sql """
+            ALTER TABLE add_keys_light_schema_change ADD COLUMN new_key_column INT default "2" AFTER city PROPERTIES ("timeout"="604800");
+            """
+        jobStateResult = waitJobFinish(""" SHOW ALTER TABLE COLUMN WHERE IndexName='add_keys_light_schema_change' ORDER BY createtime DESC LIMIT 1 """, 9)
+        assertEquals(jobStateResult[0][8], "-1")
+        sql """ 
+                INSERT INTO add_keys_light_schema_change (user_id,username,city,age,sex,phone,address,register_time) VALUES
+                (1, 'Jone', 'Beijing', 10, 1, 10010, 'Haidian', '2017-10-01 13:00:00')
+            """
+        sql """ 
+                INSERT INTO add_keys_light_schema_change (user_id,username,city,new_key_column,age,sex,phone,address,register_time) VALUES
+                (1, 'Jone', 'Beijing', 1, 10, 1, 10010, 'Haidian', '2017-10-01 14:00:00')
+            """
+        table_tablets = sql """ SHOW TABLETS FROM add_keys_light_schema_change ORDER BY RowCount DESC LIMIT 1 """
+
+        qt_41_unique_multi_rowset """ SELECT * FROM add_keys_light_schema_change; """
```

Comment:
For all test cases, you should check the compaction logic:
1. first disable compaction and run all cases.
2. call compaction API to do compaction for the table, and then run all test cases.